src/profile/notification-settings/NotificationSettings.ts

```diff
@@ -0,0 +1,184 @@
+import _ from 'lodash/fp';
+import { useEffect, useState } from 'react';
```

Comment:
Due to the large number of changes in this file, it was not recognized as a rename from `NotificationSettings.js`.